src/modules/CreateChannel/components/InviteUsers/index.tsx

```diff
@@ -91,10 +91,12 @@ const InviteUsers: React.FC<InviteUsersProps> = ({
       titleText={titleText}
       submitText={submitText}
       type={ButtonTypes.PRIMARY}
-      disabled={Object.keys(selectedUsers).length === 0}
+      disabled={users.length > 0 && Object.keys(selectedUsers).length === 0}
```

Comment:
If possible, can you add a  test for this -> 
`should enable create button when there are no users in the list` 
`should disable create button when there are users on list but none's checked` 

Separate PR is okay

Else, comments are fine too

Reply:
Because this is a bit unobvious part ~

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/665/commits/16b47590799c020590999022f86408a7a652aa9b added a comment first. Will add some unit tests separately!